Command/RemoveCacheCommand.php

```diff
@@ -20,56 +21,89 @@
 
 class RemoveCacheCommand extends Command
 {
-    /* @var CacheManager cacheManager */
-    private $cacheManager;
+    protected static $defaultName = 'liip:imagine:cache:remove';
 
-    public function __construct(CacheManager $cacheManager)
+    use CacheCommandTrait;
+
+    public function __construct(CacheManager $cacheManager, FilterManager $filterManager)
     {
-        $this->cacheManager = $cacheManager;
+        parent::__construct();
 
-        parent::__construct('liip:imagine:cache:remove');
```

Comment:
do we only allow symfony versions that support `$defaultName`? otherwise we need to pass the protected static field to the constructor.

Reply:
I think so, [it looks like it's supported since 3.4](https://github.com/symfony/symfony/pull/24609)